types/crypto-ld/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for crypto-ld 7.0
+// Project: https://github.com/digitalbazaar/crypto-ld
+// Definitions by: behruzrahimov <https://github.com/behruzrahimov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class CryptoLD {
+    constructor(params?: { suites: any });
```

Comment:
this looks to be a Map, keyed by string, as per comments in source, not `any`, at least `Map<string, unknown>` IMO

Reply:
fixed